lib/host/wasi/inode-macos.cpp

```diff
@@ -1079,16 +1079,76 @@ WasiExpect<void> INode::sockSetOpt(__wasi_sock_opt_level_t SockOptLevel,
   return {};
 }
 
-WasiExpect<void> INode::sockGetLoaclAddr(uint8_t *, uint32_t *,
-                                         uint32_t *) const noexcept {
-  return WasiUnexpect(__WASI_ERRNO_NOSYS);
-}
+WasiExpect<void> INode::sockGetLoaclAddr(uint8_t *AddressPtr,
```

Comment:
The #2103 is merged. Please rebase and fix the naming.

Reply:
Hold on. The name is still wrong :-)

Reply:
Sorry, my fault. XD